coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -107,29 +81,25 @@
               <v-expand-transition>
                 <div class="director-info" v-show="activeIndex !== index">
                   <div class="address">
-                    <div class="address__row">{{director.deliveryAddress.streetAddress}}</div>
-                    <div class="address__row">{{director.deliveryAddress.streetAddressAdditional}}</div>
-                    <div class="address__row">
-                      <span>{{director.deliveryAddress.addressCity}}</span>
-                      <span>&nbsp;{{director.deliveryAddress.addressRegion}}</span>
-                      <span>&nbsp;{{director.deliveryAddress.postalCode}}</span>
-                    </div>
-                    <div class="address__row">{{director.deliveryAddress.addressCountry}}</div>
+                    <BaseAddress v-bind:address="director.deliveryAddress"  />
+                  </div>
+                  <div class="actions">
+                    <v-btn small flat color="primary" :disabled="!agmEntered"
```

Comment:
It doesn't seem right that this component needs to know about a sibling component. It seems better that the parent (AR page) should tell THIS component whether or not it should allow editing.
Thoughts?